src/test/csfle.rs

```diff
@@ -548,9 +548,9 @@ async fn bson_size_limits() -> Result<()> {
 
     // Setup: encrypted client.
     let mut opts = get_client_options().await.clone();
-    let handler = Arc::new(EventHandler::new());
-    let mut events = handler.subscribe();
-    opts.command_event_handler = Some(handler.clone().into());
+    let buffer = EventBuffer::<Event>::new();
```

Comment:
A type annotation was required here; it's not clear to me what made this harder to infer than any of the other call sites.